src/api/index.spec.tsx

```diff
@@ -139,7 +158,7 @@ describe(__filename, () => {
       await callApiWithDefaultApiState({ endpoint: '/url', query });
 
       expect(fetch).toHaveBeenCalledWith(
-        '/api/v4/url/?foo=1&bar=abc',
+        `/api/v4/url/${defaultQueryString}&foo=1&bar=abc`,
```

Comment:
I'd rather not keep investing in this testing style for two reasons:

1. As you can see here, you had to fix every test that compared query strings even though they had nothing to do with the `lang` feature.

2. Objects are hash tables. There is [no guarantee](https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order) that their keys will be in order.

The solution we had in addons-frontend solved both of these issues and I filed https://github.com/mozilla/addons-code-manager/issues/164 to add it to the code-manager. Since this patch doubles down on what we had previously, I'd like to see the assertions switch to a [helper like this](https://github.com/mozilla/addons-frontend/blob/aabbd8097c58ae1bc5bb48bc942a63bf6bfd2e12/tests/unit/helpers.js#L909-L931) at the very least. That helper lets you only check the necessary parameters and would address both points above.

Reply:
I introduced a jest matcher to deal with this.